src/jit/importer.cpp

```diff
@@ -6462,6 +6463,11 @@ var_types Compiler::impImportCall(OPCODE                  opcode,
         call->gtCall.callSig  = new (this, CMK_CorSig) CORINFO_SIG_INFO;
         *call->gtCall.callSig = calliSig;
 #endif // DEBUG
+
+        if (IsTargetAbi(CORINFO_CORERT_ABI))
```

Comment:
@sandreenko @MichalStrehovsky 
>do you have test, that can reproduce that?
No... I'll create an issue to track this (there will probably be more work needed than just a simple hookup to the changes in this PR). I'll put in a complete description of the problem there.
>let's skip this for now
Absolutely :). Don't make any GVM related changes to this PR, and let it just be scoped to the calli case.